### ISSUE TITLE
closes when destination folder has been cleared

### ISSUE THREAD

**Issue by ghost** [Unvanquished/updater#109]
Step to reproduce on Debian:

* run the updater so that the game can work
* remove everything in the download folder, except the `unvanquished` folder
* run the updater

What happens:

A popup saying `Failed to start Unvanquished process.` shows, and the updater ends.

What would be expected:

Updater should at least ask to repair (aka: reinstall).

**Comment by ghost** [comment]
Of course, clicking on the "settings" wheel allows to force the updater to download, but I think this should be done automatically. Clicking on that wheel is "not so simple" and normal users could think they need more extreme measures.

**Comment by slipher** [comment]
Checking the full installation integrity is not feasible, but I guess we could do a minimal check that the engine binary exists

**Comment by ghost** [comment]
It could be possible to check if the daemon executable fails to start, or returns too quickly. It could also be possible to make the updater place a marker that the client would remove if successfully started. If marker is still there when updater is run anew, then show an emergency  window or something.
Just some random ideas.
